assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -340,14 +362,12 @@ class PageEdit extends Component {
 										) }
 										id="story-background-media"
 									/>
+									{ mediaUrl && (
+										<Button onClick={ () => setAttributes( { mediaUrl: undefined, mediaId: undefined, mediaType: undefined } ) } isLink isDestructive>
+											{ _x( 'Remove', 'background media', 'amp' ) }
```

Comment:
Maybe we shouldn't even show the `Remove` button anymore? Since it's set automatically anyway if missing, maybe just "Replace" is enough?

Reply:
Yup! Already got rid of that link in the video block.

Reply:
Thought: With this change we're likely always going to have a poster image - why even show the replacement option?

Reply:
Hmm. Probably some users would want to choose a poster image they like, the first frame isn't necessarily a good representation of the video. True that most users probably wouldn't change the setting but there are probably some who would want to choose that.